### PR TITLE
Ft update ansible IaC

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- [restructure ansible folder](https://github.com/airqo-platform/AirQo-api/pull/1201/commits/454d23a4030ccca71b575e1bd08be46f510167d4)
- Adds [ansible compute disk](https://github.com/airqo-platform/AirQo-api/pull/1201/commits/d2957e4cbb6727cb16baf62084bf0bb9742f9c27)
- Adds [ansible instance](https://github.com/airqo-platform/AirQo-api/pull/1201/commits/dcf2652db83b2c7f08e981626ea4e1cb4377db90)

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
 - [PLATOPS-29](https://airqoteam.atlassian.net/browse/PLATOPS-29?atlOrigin=eyJpIjoiYTAxNzQ1NTExNDBhNGFlZmFkZWQ4NmUwMjhhMzhiOGYiLCJwIjoiaiJ9)

**_HOW DO I TEST OUT THIS PR?_**
- Do a static code analysis

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
- n/a

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
- n/a

**_ARE THERE ANY RELATED PRs?_**
- n/a

[PLATOPS-29]: https://airqoteam.atlassian.net/browse/PLATOPS-29?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: infra/terraform/gcp/modules/ComputeInstance/ansible-controller.tf [hunk 1]
```diff
@@ -0,0 +1,43 @@
+resource "google_compute_instance" "ansible_controller" {
+  boot_disk {
+    auto_delete = true
+    source = "ansible-controller"
+  }
+
+  machine_type = "e2-small"
+
+  metadata = {
+    startup-script = "sudo ufw allow ssh"
```
Comment (123MwanjeMike) [inline infra/terraform/gcp/modules/ComputeInstance/ansible-controller.tf:10]: Ansible controller instance

File: infra/terraform/gcp/modules/ComputeDisk/ansible-controller.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+resource "google_compute_disk" "ansible_controller" {
+  image                     = var.os["ubuntu-focal"]
+  name                      = "ansible-controller"
+  physical_block_size_bytes = 4096
+  project                   = var.project-id
+  size                      = var.disk_size["small"]
+  type                      = "pd-standard"
+  zone                      = var.zone
+}
+# terraform import google_compute_disk.ansible_controller projects/${var.project-id}/zones/europe-west1-b/disks/ansible-controller
```
Comment (123MwanjeMike) [inline infra/terraform/gcp/modules/ComputeDisk/ansible-controller.tf:10]: Ansible controller disk

File: infra/ansible/README.md [hunk 3]
```diff
@@ -2,31 +2,10 @@
...
-
-    extension/
-        setup/                 # All the setup files for updating roles and ansible dependencies
+    playbooks/              # All playbooks go here
+    setup/                  # Files here are added to the ansible control node
+        etc/                # Files here go to the '/etc/ansible/' path
+            roles/          # Directory for all roles
+            ansible.cfg     # Ansible.cfg file that holds all ansible configurations
+            hosts           # Static inventory file
+        gcp.yaml            # This GCP dynamic inventory file goes to the '/opt/ansible/inventory/gcp.yaml' path
```
Comment (123MwanjeMike) [inline infra/ansible/README.md:11]: Update README

**Review by 123MwanjeMike — COMMENTED** [review]
Guiding comments

**Review by Baalmart — APPROVED** [review]
LGTM, thanks Mike!

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**